akka-cluster-typed/src/main/scala/akka/cluster/typed/internal/receptionist/ClusterReceptionist.scala

```diff
@@ -90,6 +94,10 @@ private[typed] object ClusterReceptionist extends ReceptionistBehaviorProvider {
     def selfUniqueAddress: UniqueAddress = cluster.selfUniqueAddress
   }
 
+  def uniqueKey(key: ServiceKey[_], service: ActorRef[_]): String = {
+    key.toString + service.path.toStringWithoutAddress
+  }
```

Comment:
Style choice but several functions do not use `{}` and this doesn't appear to benefit from them.

Reply:
Yep, agree, this should be without curlies